example/workflows/cnm.yml

```diff
@@ -25,6 +25,8 @@ KinesisTriggerTest:
           Next: CnmResponseFail
       Next: TranslateMessage
     TranslateMessage:
+      envs:
```

Comment:
So I'm guessing you tested and this actually works? Usually we pass envs to the lambda in the lambda definition and I don't see `envs` in the Amazon State Language.

Reply:
No, this shouldn't have been put in review until the client lib passed and the CNM code is updated.